.vib/drupal-nginx/goss/drupal-nginx.yaml

```diff
@@ -0,0 +1,31 @@
+file:
+  # HTTP vhost should have been properly rendered
+  /opt/bitnami/nginx/conf/server_blocks/drupal-server-block.conf:
+    exists: true
+    filetype: file
+    contains:
+      - /root\s+/opt/bitnami/drupal/
+  # HTTPs vhost should have been properly rendered
+  /opt/bitnami/nginx/conf/server_blocks/drupal-https-server-block.conf:
+    exists: true
+    filetype: file
+    contains:
+      - /listen \d+ ssl/
+      - /root\s+/opt/bitnami/drupal/
+  /opt/bitnami/drupal/sites/default/settings.php:
+    exists: true
+    filetype: file
+    contains:
+      - "$settings['trusted_host_patterns'] = array('^.*$');"
+  /opt/bitnami/php/etc/php.ini:
+    exists: true
+    filetype: file
+    contains:
+      - /^memory_limit = 256M/
+  /opt/bitnami/nginx/conf/bitnami/php-fpm.conf:
+    exists: true
+    filetype: file
+    contains:
+      - "index index.html index.htm index.php;"
+      - "location ~ \\.php$|^/update.php {"
+      - "fastcgi_index  index.php;"
```

Comment:
This section can't be moved to a common `php-fpm` library because it is part of Drupal-nginx postunpack customization:
https://github.com/bitnami/containers/blob/15cbce818b983c94a9145ff27b720bf1d99e33ea/bitnami/drupal-nginx/10/debian-11/rootfs/opt/bitnami/scripts/drupal/postunpack.sh#L120-L124